packages/backend/src/controllers/validationController.ts

```diff
@@ -34,12 +35,17 @@ export class ValidationController extends Controller {
     async post(
         @Path() projectUuid: string,
         @Request() req: express.Request,
+        @Query() context?: ValidateProjectPayload['context'],
```

Comment:
We can use the existing header information to detect if a request comes from the CLI, APP, other. This header is available in all requests.

Example in the projectRouter
```getRequestMethod(req.header(LightdashRequestMethodHeader)),```

With the controller, it should be possible to have something like:
```
@Header(LightdashRequestMethodHeader) context?: string;
...

jobId: await validationService.validate(
                    req.user!,
                    projectUuid,
                    context: getRequestMethod(context),
                ),
```

Reply:
good idea ! I'll do it this way